corecito.py

```diff
@@ -47,26 +48,38 @@ async def main():
       decrease_percentage = missing * 100 / account.core_number
 
       if coreNumberExploded(account.core_number, deviated_core_number, account.max_core_number_increase_percentage):
-        logger.info(f'> Exploded {increase_percentage:.2f}%\nConsider updating CoreNumber to {deviated_core_number:.6f}')
+        log_message = f'> Exploded {increase_percentage:.2f}%\nConsider updating CoreNumber to {deviated_core_number:.6f}'
+        logger.info(log_message)
+        if config['telegram_notifications_on']:
+          telegram_bot_sendtext(config['telegram_bot_token'], config['telegram_user_id'], log_message)
 
       elif coreNumberIncreased(account.core_number, deviated_core_number, account.min_core_number_increase_percentage, account.max_core_number_increase_percentage):
         logger.info(f'Increased {increase_percentage:.2f}% - excess of {excess:.6f} {account.core_number_currency} denominated in {account.base_currency}')
         tx_result = round(excess * buy_price, account.max_decimals_buy)
-        logger.info(f'\n\n>>> Selling: {tx_result:.6f} {account.base_currency} at {buy_price} to park an excess of {excess:.6f} {account.core_number_currency}\n')
+        log_message = f'\n\n>>> Selling: {tx_result:.6f} {account.base_currency} at {buy_price} to park an excess of {excess:.6f} {account.core_number_currency}\n'
+        logger.info(log_message)
         # Sell excess of base currency ie. => in ETH_BTC pair, sell excess BTC => Buy ETH
         if (not config['safe_mode_on']):
           await account.order_market_buy(tx_result, excess)
+          if config['telegram_notifications_on']:
+              telegram_bot_sendtext(config['telegram_bot_token'], config['telegram_user_id'], log_message)
```

Comment:
Same question about moving these logs messages to a separate function and also the other 2 branches of checks

Reply:
I think so, we can even move the log messages to another file so to get rid of all the text and leave the code cleaner. I will work on an example on this PR itself and then we can see if it makes better sense.